### DISCUSSION TITLE
Problems with motherboard sensors

### DISCUSSION THREAD

**Discussion by Cheesekeeper** [devcompl/TTController#119] [Configuration] [answered]
Hi, another question if I may.  Trying to modify my configuration to use one of the temperature sensors  on the motherboard to control fan speeds.  For some reason, while all of the motherboard sensors are visible in the debug report (with correct values), it doesn't seem to find them when I use them in the config.  I am trying to use sensor EC_TEMP1, but get the same error with all sensors on the motherboard.  No problems when using the CPU or GPU sensors.

Excerpt from config.json:
`    {
      "Sensors": ["/lpc/it8688e/temperature/1"],
      "Config": {
        "CriticalValue": 50
      }
	}`

Excerpt from report:

```
Sensors

»
»  ITE IT8688E:
»        Temperature:
»                System #1 (/lpc/it8688e/temperature/0): 40.0 °C
»                EC_TEMP1 (/lpc/it8688e/temperature/1): 29.0 °C
»                CPU (/lpc/it8688e/temperature/2): 55.0 °C
»                PCIe x16 (/lpc/it8688e/temperature/3): 52.0 °C
»                VRM MOS (/lpc/it8688e/temperature/4): 40.0 °C
»                PCH (/lpc/it8688e/temperature/5): 48.0 °C
»
»  ITE IT8792E/IT8795E:
»        Temperature:
»                PCIe x8 (/lpc/it879xe/temperature/0): 43.0 °C
»                System #2 (/lpc/it879xe/temperature/2): 38.0 °C
»
»  AMD Ryzen 9 3950X 1:
»        Temperature:
»                Core (Tctl/Tdie) (/amdcpu/0/temperature/2): 55.6 °C
»                CCD1 (Tdie) (/amdcpu/0/temperature/3): 47.3 °C

```

Output from service:
`19:42:13.4783: Assigned plugin "PwmSpeedController" to "CPUFanDefault" profile
19:42:13.4783: Could not find "/lpc/it8688e/temperature/1" sensor identifier!
19:42:13.4783: Could not find "/lpc/it8688e/temperature/1" sensor identifier!`
 
Any suggestions?  Thanks....

**Comment by MoshiMoshi0** [comment, marked as answer]
By default almost every type of sensor is disabled for performance: https://moshimoshi0.github.io/TTController/config/service#motherboardsensorsenabled

**Comment by Cheesekeeper** [comment]
Thank you :)  Missed that one.....
